rollup.config.js

```diff
@@ -28,7 +28,9 @@ const { packageJson: pkg } = readPkgUp.sync({
 });
 const [, packageName] = pkg.name.split('@commercetools-frontend/');
 const extensions = ['.js', '.ts', '.tsx'];
-const babelOptions = getBabelPreset();
+const babelOptions = getBabelPreset(null, {
+  runtime: 'automatic',
```

Comment:
I am not super sure here. This was sort of the initial reason for the soft opt-in. We could also bundle with `classic`. This would assume that any newly published version runs with a recent React version I'd assume.

Reply:
The last CI run is interesting. It means you cannot have automatic and import React. Either we bundle classic but I am not sure if that would break consumers using automatic then again.